metadata/app/schema.go

```diff
@@ -79,23 +79,19 @@ type EnvironmentSpecs map[string]*EnvironmentSpec
 // KubernetesVersion: The Kubernetes version the target cluster is running on.
 // Path:              The relative path containing metadata for this environment,
 //                    rooted at the directory 'environments'.
-// Destinations:      One or more cluster addresses that this environment
-//                    points to.
+// Destination:       Cluster address that this environment points to.
```

Comment:
These comments should be in the type definition so they show up properly with godoc. Currently there will be docs on the types, but not the members. The same thing applies to `EnvironmentDestinationSpec` as well.